interface/src/app/node/node_search.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * "adage.node_search" module.
+ */
+
+angular.module('adage.node_search', [
+  'ui.router',
+  'ui.bootstrap',
+  'ngResource',
+  'adage.mlmodel.components',
+  'adage.utils'
+])
+
+.config(['$stateProvider', function($stateProvider) {
+  $stateProvider.state('node_search', {
+    url: '/node/search',
+    views: {
+      main: {
+        templateUrl: 'node/node_search.tpl.html',
+        controller: 'NodeSearchCtrl'
+        // See comments below on why "controllerAs" is not used.
+      }
+    },
+    data: {pageTitle: 'Node Search'}
+  });
+}])
+
+.factory('Node', ['$resource', 'ApiBasePath',
```

Comment:
Hmm are they really different though? It looks like the NodeInfo one just has the ``:id`` parameter for requests that are non-GET. It also looks like it does not use the ApiBasePath, which it probably should, right? Wouldn't it make sense to just keep one of them?